enterprise/server/remote_execution/runner/runner.go

```diff
@@ -83,6 +85,11 @@ const (
 	hitStatusLabel = "hit"
 	// Label assigned to runner pool request count metric for unfulfilled requests.
 	missStatusLabel = "miss"
+
+	// Value for persisent workers that support the JSON persistent worker protocol.
+	workerProtocolJSONValue = "json"
+	// Value for persisent workers that support the protobuf persistent worker protocol.
+	workerProtocolProtobufValue = "protobuf"
```

Comment:
Since the values for `requires-worker-protocol` are `json` and `proto`, and `WorkerProtocolFormat` is `JSON` and `PROTO`, I say we should go with `proto` here. I know it bucks the current design doc, but afaik no one else has implemented this side of it, and I'll comment in it with this same reasoning.

Reply:
I updated my bazel patch here: https://github.com/brentleyjones/bazel/tree/5.0/remote-persistent-workers

Reply:
Agreed - done.